app/App.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace app;
+
+class App
+{
+    protected $container;
+
+    public function __construct()
+    {
+        $this->container = new Container([
```

Comment:
ovako nikada nećeš moći koristiti neki drugi container, a da ne editiraš `App` klasu. Ja bi to ovako napravio:

```suggestion
        public function __construct(Container $container)
        {
            $this->container = $container;
        }
```

Novu instancu `Container`a bi napravio tamo gdje instanciraš i `App`